src/playgroundrl_envs/games/open_spiel/generic.py

```diff
@@ -0,0 +1,79 @@
+import json
+import pyspiel
+
+class OpenSpielGame:
+    def __init__(self, game_name):
```

Comment:
Instead of passing in the `game_name` directly, we can pass in a `OpenSpielParameters` class that extends `GameParameterInterface`. In general, the signature of `__init__` should match that of `PettingZooGame`. 

When we use the game_parameter interface, we can then very easily pass in a different game_name from the client, or the frontend.